hpat/tests/test_dataframe.py

```diff
@@ -606,6 +606,18 @@ def test_impl():
             # restore global val
             hpat.hiframes.sort.MIN_SAMPLES = save_min_samples
 
+    def test_df_isna1(self):
+        def test_impl(df):
+            return df.isna()
+        hpat_func = hpat.jit(test_impl)
+
+        # TODO: add column with datetime values when test_series_datetime_isna1 is fixed
+        df = pd.DataFrame({'A': [1.0, 2.0, np.nan, 1.0],
+                           'B': [np.nan, 5, 6, np.nan],
```

Comment:
it might be good to add `np.inf` here also

Reply:
Added